Package.swift

```diff
@@ -17,6 +17,7 @@ let package = Package(
 		),
 	],
 	dependencies: [
+		// TODO: .package(name: "secp256k1", url: "https://github.com/GigaBitcoin/secp256k1.swift", .upToNextMinor(from: "0.10.0")),
```

Comment:
We can't bring these back because of cocoapod issues. Can you remove these todos?